app/jobs/set_allele_registry_ids.rb

```diff
@@ -0,0 +1,25 @@
+require 'rbconfig'
+
+class SetAlleleRegistryIds < AlleleRegistryIds
+  def perform(recurring = true)
+    begin
+      for v in Variant.where(allele_registry_id: nil) do
```

Comment:
I didn't actually realize you could do this! `.each` would certainly be more idiomatic, but if this works, I don't have a problem with it.

Reply:
It works but it is actually not recommended. I think I was a bit out in Python-land when I wrote it. I will change it to the more idiomatic version.